migtests/scripts/functions.sh

```diff
@@ -132,7 +160,23 @@ run_sqlplus() {
 	db_name=$1
 	sql=$2
 	conn_string="${SOURCE_DB_USER}/${SOURCE_DB_PASSWORD}@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}"
-	echo exit | sqlplus -f ${conn_string} @${sql}
+	echo exit | sqlplus -f "${conn_string}" @"${sql}"
+}
+
+
+run_sqlplus_grants() {
+	db_name=$1
+    local file_name=$2
+    conn_string="sys/password@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name} as SYSDBA"
+	echo exit | sqlplus -f "${conn_string}" @"${file_name}"
+}
+
+
+run_sqlplus_write() {
+    db_name=$1
+    sql=$2
+    conn_string="test_user/password@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}"
+    echo exit | sqlplus -f "${conn_string}" @"${sql}"
```

Comment:
Just the way we did with PG, let's have two functions: run_sqlplus() and run_sqlplus_admin().
The first should connect with $SOURCE_DB_USER and $SOURCE_DB_PASSWORD. And the later should connect using $SOURCE_DB_ADMIN_USER and $SOURCE_DB_ADMIN_PASSWORD.